--- gst-libav-1.14.2.orig/ext/libav/Makefile.am	2018-09-09 14:03:44.854038903 +0200
+++ gst-libav-1.14.2/ext/libav/Makefile.am	2018-09-09 14:05:02.174150234 +0200
@@ -22,7 +28,7 @@
 libgstlibav_la_LIBADD = $(GST_PLUGINS_BASE_LIBS) \
 	-lgstaudio-$(GST_API_VERSION) -lgstvideo-$(GST_API_VERSION) \
 	-lgstpbutils-$(GST_API_VERSION) $(GST_BASE_LIBS) \
-	 $(LIBAV_LIBS) $(WIN32_LIBS) -lz $(BZ2_LIBS) $(LZMA_LIBS)
+	 $(LIBAV_LIBS) $(WIN32_LIBS) -lz $(BZ2_LIBS) $(LZMA_LIBS) -lvdpau -lX11
 libgstlibav_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS) $(DARWIN_LDFLAGS)
 
 if HAVE_LIBAV_UNINSTALLED
