--- gst-libav-1.6.1/configure.ac.crystalhd	2015-12-10 22:05:00.805989084 +0100
+++ gst-libav-1.6.1/configure.ac	2015-12-10 22:15:19.024305997 +0100
@@ -241,6 +241,11 @@ if test "x$HAVE_LZMA" = "xno"; then
     AC_WARN([lzma not found, tiff reader will not be able to read lzma files])
 fi
 
+dnl *** Check for crystalhd
+AC_MSG_CHECKING([for crystalhd])
+AC_CHECK_HEADERS([libcrystalhd/libcrystalhd_if.h],[HAVE_CRYSTALHD=yes],[HAVE_CRYSTALHD=no])
+AM_CONDITIONAL(HAVE_CRYSTALHD, test "x$HAVE_CRYSTALHD" = "xyes")
+
 AC_ARG_ENABLE(gpl,
               [AC_HELP_STRING([--enable-gpl], [build a GPL licensed gst-libav])])
 AM_CONDITIONAL(GST_LIBAV_ENABLE_GPL, test "x$enable_gpl" = "xyes")
--- gst-libav-1.6.1/ext/libav/Makefile.am.crystalhd	2015-12-10 22:15:53.568283264 +0100
+++ gst-libav-1.6.1/ext/libav/Makefile.am	2015-12-10 22:16:45.042575294 +0100
@@ -12,6 +12,12 @@ else
     LZMA_LIBS =
 endif
 
+if HAVE_CRYSTALHD
+    CRYSTALHD_LIBS = -lcrystalhd
+else
+    CRYSTALHD_LIBS =
+endif
+
 libgstlibav_la_SOURCES = gstav.c	\
 			  gstavprotocol.c	\
 			  gstavcodecmap.c	\
@@ -33,7 +39,7 @@ libgstlibav_la_CFLAGS = $(LIBAV_CFLAGS)
 libgstlibav_la_LIBADD = $(LIBAV_LIBS) $(GST_PLUGINS_BASE_LIBS) \
 	-lgstaudio-$(GST_API_VERSION) -lgstvideo-$(GST_API_VERSION) \
 	-lgstpbutils-$(GST_API_VERSION) $(GST_BASE_LIBS) \
-	$(LIBM) $(WIN32_LIBS) -lz $(BZ2_LIBS) $(LZMA_LIBS)
+	$(LIBM) $(WIN32_LIBS) -lz $(BZ2_LIBS) $(LZMA_LIBS) $(CRYSTALHD_LIBS)
 libgstlibav_la_LDFLAGS = $(GST_PLUGIN_LDFLAGS) $(DARWIN_LDFLAGS)
 libgstlibav_la_LIBTOOLFLAGS = $(GST_PLUGIN_LIBTOOLFLAGS)
 
